classes/cliTool/CliTool.inc.php

```diff
@@ -81,4 +89,56 @@ function __construct($argv = array()) {
 	function usage() {
 	}
 
+	private function checkArgsForUsername() {
+		$usernameKeyPos = array_search('--user_name', $this->argv);
```

Comment:
Can we support `--user_name` and `-u`? Just to make it a little quicker...

Reply:
Fixed